.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Please perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Aubccm%2Fubc-def) before reporting an issue.**
+
+**Describe the bug in detail**
```

Comment:
I t looks like the linebreak formatting isn't applying - https://github.com/UBCCM/UBC-DEF/blob/e5904c8a6be5e1e9012a18ddf8729b9702710e44/.github/ISSUE_TEMPLATE/bug_report.md

Two spaces at the end of each of the bolded lines should do it.

Reply:
Updated.